PyIlmBase/PyImathNumpy/CMakeLists.txt

```diff
@@ -1,36 +1,47 @@
+if(TARGET Python2::Python AND
```

Comment:
yikes, this is complex, I'm looking forward to pybind11

Reply:
yes, boost, and boost python is a nightmare, and then numpy is annoying because it's not part of boost python until 1.63. Python is a bit complex handling 2 and 3 at the same time, but that's a useful thing for VFX Platform 2020...